src/components/Challenges/CodeChallenge/CodeChallenge.tsx

```diff
@@ -46,6 +46,15 @@ export const CodeChallenge = ({
 
   useEffect(resetChallenge, [id]);
 
+  // passes test when control & enter keys are pressed together
+  const passTestsOnKeypress = () => {
+    window.addEventListener('keypress', (e:KeyboardEvent) => {
+      ((e.ctrlKey || e.metaKey) && e.code === "Enter") ? runTests() : null;
```

Comment:
keep it as a regular if statement not as a ternary, it may be slightly shorter this way but it's harder to read. Shorter is often not better when it means being unconventional in coding